README.md

```diff
@@ -227,7 +227,7 @@ Note that you can also use the [Rendertron middleware](https://github.com/Google
 
 ## Google App Engine Quickstart
 
-[Google App Engine](https://cloud.google.com/appengine/) is a managed server platform that [supports Node](https://cloud.google.com/nodejs/) in its [Flexible Environment](https://cloud.google.com/appengine/docs/flexible/). You can deploy prpl-server to App Engine with a few steps:
+[Google App Engine](https://cloud.google.com/appengine/) is a managed server platform that [supports Node](https://cloud.google.com/nodejs/) in its [flexible environment](https://cloud.google.com/appengine/docs/flexible/nodejs/) or [standard environment](https://cloud.google.com/appengine/docs/standard/nodejs/). You can deploy prpl-server to App Engine with a few steps:
```

Comment:
I'd say let's only mention standard to keep it simple, also mention that standard has a free tier?

Reply:
Standard is not necessarily free - you need to attach a billing account and you'll be billed if you go over quota (28 instance hours/day iirc). Also FWIW, standard is currently in beta.

Reply:
Free tier is the terminology they use, so I think we should mention it because its relevant to being low friction getting started. https://cloud.google.com/free/docs/always-free-usage-limits. 

IMO a public beta is stable enough for us to recommend for this kind of example.